azure-pipelines.yml

```diff
@@ -1341,6 +1278,169 @@ jobs:
       includeRootFolder: false
       sevenZipCompression: 5
       archiveFile: $(Build.ArtifactStagingDirectory)/zip/lpk-linux-arm64.zip
+  
+  - task: PublishBuildArtifacts@1
+    displayName: Generate lks.json for .zip files
+    inputs:
+      PathtoPublish: $(Build.ArtifactStagingDirectory)/zipv2
```

Comment:
Could you help me understand what's happening in this step? The `displayName` doesn't seem to reflect that it's publishing something - is that a copy-paste thing?

Also, I can't see anything being built in the zipv2 directory prior to this publishing step, so I can't tell what's being published.

Reply:
I have renamed the displayName, yes this task publishes or stores the generated archives/zip files for zipv2 in the zipv2 directory in the build systems, which will be used later during the release process. Release pipeline is [here](https://devdiv.visualstudio.com/DevDiv/_releaseProgress?_a=release-environment-logs&releaseId=1500723&environmentId=8176210)